docs/reference/2-ods-api/how-to-guides/how-to-create-a-custom-populated-template.md

```diff
@@ -215,7 +176,7 @@ already been created. You can also create a new one by running the
 or by running `nuget spec`.
 
 The nuspec file created by the `New-PopulatedTemplateNuspec` script will be
-created at `<source directory>`\\Ed-Fi-ODS-Implementation\\DatabaseTemplate\\Database\\Populated.Template.nuspec.
+created at `<source directory>\\Ed-Fi-ODS-Implementation\\DatabaseTemplate\\Database\\Populated.Template.nuspec`.
```

Comment:
There are probably more weird export mistakes like this that will need to be fixed in the future.